src/app/LoggedInOutlet.ts

```diff
@@ -26,6 +24,6 @@ export class LoggedInRouterOutlet extends RouterOutlet {
     if (!this.publicRoutes[url] && !localStorage.getItem('jwt')) {
       instruction.component = Login;
     }
-    super.activate(instruction);
+    return super.activate(instruction);
```

Comment:
you must `return` here otherwise you will have an error `TypeError: Cannot read property 'then' of undefined`